Documentation/Architecture/CrossPlatform/DeterminingPlatformCapabilities.md

```diff
@@ -0,0 +1,60 @@
+# Determining platform capabilities
+
+A common question asked of the MRTK involves knowing which specific device (ex: Microsoft HoloLens 2) is being
+used to run an application. The goal behind that knoweldge generally centers around knowing whether or not a
```

Comment:
Nit: I'm not sure this second sentence adds that much - it might be worth rewording this first section to like:

A common question asked of the MRTK involves knowing which specific device (ex: Microsoft HoloLens 2) is being
used to run an application. Identifying the exact hardware can be challenging on different platforms. The MRTK
instead provides a way identify specific capabilities of the system (for example, if the system has the articulated
hands capability)

Reply:
i like that!